src/main/java/NuSchedule.java

```diff
@@ -62,7 +62,8 @@ public void run() {
             try {
                 String fullCommand = ui.readCommand();
                 ui.printLine(); // show the divider line ("_______")
-                Command c = Parser.parse(fullCommand, locations);
+                int size = events.getSize();
```

Comment:
This is the second time you add size into the main run(). We have discussed that, for OOP, you should not access events methods in run(). You may want to call this function in EditCommand which allows you to check the size.